.github/workflows/custom-branch-builds.yml

```diff
@@ -0,0 +1,50 @@
+name: Custom Branch Builds 
+
+on: 
+  workflow_dispatch:
+  
+    inputs:
+      buildosflavor:
+        description: 'Select the OS Flavour - centos7 (Only Supported) '
+        required: true
+        options: ['centos7']
+        default: 'centos7'
+     
+      branchesflavor:
+        description: 'Select the Gluster Branch: eg:- devel, v10.1, v9.5,v8.6'
```

Comment:
```suggestion
        description: 'Select the Gluster Branch: eg:- devel, release-10, release-9'
```

Reply:
This will not select a particular release version. Else we need to give another argument for the subversion in a release. What is taken is the tag for the branch which is easy to co-relate.